src/styles.css

```diff
@@ -2,15 +2,51 @@
   font-family: sans-serif;
   text-align: center;
 }
-.homeview-user-names    {
-    font-family: Arial, Helvetica, sans-serif;
-    text-transform: capitalize;
+
+.homeview-user-names {
+  font-family: Arial, Helvetica, sans-serif;
+  text-transform: capitalize;
 }
 
-a:link, a:visited {
+a:link,
```

Comment:
The BEM naming conventions have to be applied to all of the code within master branch. I will create a separate issue for this.